test/Microsoft.Azure.Devices.Edge.Test/PriorityQueues.cs

```diff
@@ -39,34 +39,17 @@ public async Task PriorityQueueModuleToModuleMessages()
             string trackingId = Guid.NewGuid().ToString();
             TestInfo testInfo = this.InitTestInfo(5, 1000, true);
 
-            Action<EdgeConfigBuilder> addInitialConfig = this.BuildAddInitialConfig(trackingId, trcImage, loadGenImage, testInfo);
+            string trcIPAddress = Dns.GetHostEntry(TrcModuleName).AddressList[0].ToString();
```

Comment:
Are you sure you can resolve container ip from the host?  Also when you are running this line, container hasn't been deployed yet, right?